.editorconfig

```diff
@@ -0,0 +1,8 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
+ij_kotlin_imports_layout = *
+ij_kotlin_packages_to_use_import_on_demand=com.amazon.ionelement.**
```

Comment:
I confess I don't understand what this does, even after googling it.

Reply:
```
end_of_line = lf
```
This part tells your IDE and Ktlint that the "end of line character" must be `\n` rather than `\r\n`, `\n\r`, or anything else.
```
insert_final_newline = true
```
This part says that your IDE and Ktlint should expect every file to end with a blank line. (This is not super important, but it's nice to make it consistent so you don't have PR diffs with blank lines being added and removed.)
```
ij_kotlin_imports_layout = *
```
This makes intellij (and Ktlint) sort the imports in ascii order, without any special grouping.

```
ij_kotlin_packages_to_use_import_on_demand=com.amazon.ionelement.**
```
This tells intellij (and Ktlint) that you can use a wildcard import only for `com.amazon.ionelement` and child packages. I know wildcard imports can be a controversial topic, and I think a nice compromise here is to allow wildcard imports of things that are in the Ion Element Kotlin project and to not allow wildcard imports for things that come from outside of this project.